src/os_install.c

```diff
@@ -31,6 +31,7 @@ static const struct option prog_opts[] = {
 	{ "force", no_argument, 0, 'x' },
 	{ "bundles", required_argument, 0, 'B' },
 	{ "version", required_argument, 0, 'V' },
+	{ "manifest", required_argument, 0, 'm' },
```

Comment:
Why are you adding -m to os-install? we are using -V for that. Since it is a "new" command I don't think we need to support an old flag. If people want to use the old -m flag then they can use "verify --install -m"

Reply:
I think that diagnose, repair and os-install should be consistent. -m was added to diagnose and repair. So I added it here too.
They aren't listed in docs, like diagnose and repair.